scripts/es2k/setup_infra.sh

```diff
@@ -62,13 +52,13 @@ function install_drivers () {
   modprobe mdev
   modprobe vfio-pci
   modprobe vfio_iommu_type1
-  # change this with insmod in case of new idpf driver
+  # change this with insmod in case of new idpf host driver
   modprobe idpf
```

Comment:
Wouldn't running 'make install' after building idpf driver install it in standard path and modprobe would work in that case? This should be the standard procedure. Hence, I think we should remove line 55 as this is primarily used for our testing.

Reply:
Technically yes. Out of tree drivers would install and overwrite what's in tree. If you do not want it to overwrite. IDPF is still adding features so wanted to indicate that in-tree and out-of-tree have different code.